### ISSUE TITLE
IconicsImageView doesn't reflect iiv_color

### ISSUE THREAD

**Issue by mtrakal** [mikepenz/Android-Iconics#322]
`androidIconics = '2.9.1' // https://github.com/mikepenz/Android-Iconics`
core + views

```
<com.mikepenz.iconics.view.IconicsImageView
                android:id="@+id/icon_email"
                android:layout_width="50dp"
                android:layout_height="50dp"
                android:background="#29d09e"
                android:padding="15dp"
                app:iiv_color="@color/white"
                app:iiv_icon="ANY_ICON"/>
```

Icon color is black, not white.
Before we used `2.8.9` and it works as expected. Must be regression.

**Comment by Rainer-Lang** [comment]
Try the newest for views:
compile "com.mikepenz:iconics-views:2.9.2@aar"

**Comment by mtrakal** [comment]
Some icons are back white, but when icon is set by code not. Previously was working ok.

FIY: info that version of view/core is not much visible :/. 
On https://github.com/mikepenz/Android-Iconics I look just on Maven central icon for latest version and when changed, open: https://github.com/mikepenz/Android-Iconics/releases
There is no info about different version of view/core. So maybe some info in main page?

` <color name="white_alpha_80p">#ccffffff</color>`

```
<com.mikepenz.iconics.view.IconicsImageView
                android:id="@+id/icon"
                android:layout_width="10dp"
                android:layout_height="10dp"
                android:layout_marginRight="4dp"
                android:layout_marginEnd="4dp"
                app:iiv_color="@color/white_alpha_80p"
                app:iiv_icon="moo-moon-bank1"/>
```
after is called
```
IconicsImageView icon= (IconicsImageView) view.findViewById(R.id.icon);
icon.setIcon(SOME_ICON);
```

**Comment by Rainer-Lang** [comment]
Look at the readme - and try the view-2.9.2 version

**Comment by mtrakal** [comment]
As i wrote in previous comment:

> Some icons are back white, but when icon is set by code not. Previously was working ok.

it was related to view-2.9.2 :)


```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical"
    tools:context="cz.mtrakal.iconics292.MainActivity">

    <com.mikepenz.iconics.view.IconicsImageView
        android:id="@+id/vIIVBroken"
        android:layout_width="40dp"
        android:layout_height="40dp"
        app:iiv_color="@color/colorAccent"
        app:iiv_icon="gmd-bubble-chart" />

    <com.mikepenz.iconics.view.IconicsImageView
        android:id="@+id/vIIVOk"
        android:layout_width="40dp"
        android:layout_height="40dp"
        app:iiv_color="@color/colorAccent"
        app:iiv_icon="gmd-bubble-chart" />
</LinearLayout>
```

```
package cz.mtrakal.iconics292

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.mikepenz.google_material_typeface_library.GoogleMaterial
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        vIIVBroken.setIcon(GoogleMaterial.Icon.gmd_access_time)

        vIIVOk.setIcon(GoogleMaterial.Icon.gmd_access_time)
        vIIVOk.setColorRes(R.color.colorAccent)
    }
}
```

```
    implementation "com.mikepenz:iconics-core:2.9.1@aar"
    implementation "com.mikepenz:iconics-views:2.9.2@aar"
    implementation 'com.mikepenz:google-material-typeface:3.0.1.1.original@aar'
```


![device-2017-08-22-122319](https://user-images.githubusercontent.com/1458891/29561161-bdfdecd0-8734-11e7-9d01-af8e6ea73952.png)

Maybe I'm wrong, but I think, that in our code and previous version of lib first icon have `colorAccent` and icon `gmd_access_time` after call `vIIVBroken.setIcon`.
Only set icon, but color was set from iconicsImageView from xml when we didn¨t touch on `setColor/setColorRes`. For now color is reset to default.

There is no info in readme about programatically set icons (we don't use Inflaters for automatic icon detection, so no changes for us with `LayoutInflaterCompat.setFactory2`).

**Comment by mtrakal** [comment]
When i change lines to:
```
findViewById<IconicsImageView>(R.id.vIIVOk).setColorRes(R.color.colorAccent)
findViewById<IconicsImageView>(R.id.vIIVOk).setIcon(GoogleMaterial.Icon.gmd_access_time)
```
Icon is back BLACK and not `colorAccent`...

**Comment by zTrap** [comment]
@mtrakal That bug was fixed in this [commit ](https://github.com/mikepenz/Android-Iconics/commit/7d8bdd98fe1b1aa4e0bf8914e89444730dfaf815). Please wait new version

**Comment by mikepenz** [comment]
Released v2.9.3 which should fix this as mentioend by @zTrap
